os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/src/hci_process.c

```diff
@@ -22,7 +22,7 @@ extern void hci_normal_start(void);
 extern bool bt_check_iqk(void);
 extern bool hci_start_iqk(void);
 extern uint8_t  hci_tp_lgc_efuse[0x20];
-extern uint8_t  hci_tp_phy_efuse[16];
+extern uint8_t  hci_tp_phy_efuse[19];

```

Comment:
How about defining the value? Then it is easy to maintain.

Reply:
We tend to retain the same, is that okay?
But if insist, we can change it.

Reply:
It's chipset-vendor code. You can do as you want (It's up to you). This is just suggestion for readability, but we have no plan to check and to modify this code.

Reply:
Thank you for the suggestion.
However, we will remain as it is.